From c4acdf165ac61bce88cd4264897c8783665b0799 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Mon, 13 Mar 2017 16:50:47 -0400
Subject: [PATCH 3/3] Nerf SMBIOS functions out of fwupdate

RHEL 7.4 doesn't have libsmbios, so we can't actually use it to do
anything.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 linux/Makefile | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/linux/Makefile b/linux/Makefile
index 5122e48..45bd156 100644
--- a/linux/Makefile
+++ b/linux/Makefile
@@ -8,12 +8,6 @@ LIB_LIBS= pthread
 BIN_LIBS= popt pthread
 PKLIBS	= efivar efiboot
 HAVE_LIBSMBIOS=no
-ifeq ($(ARCH),x86_64)
-	HAVE_LIBSMBIOS=yes
-endif
-ifeq ($(ARCH),ia32)
-	HAVE_LIBSMBIOS=yes
-endif
 CFLAGS	?= -g -O0
 PJONES  =
 BIN_CCLDFLAGS = $(foreach lib,$(BIN_LIBS),-l$(lib)) \
-- 
2.9.3

